### PR TITLE
Hotfix/fixed error test

### PR DESCRIPTION
**Pull request by alejandrojnm**

### CODE REVIEW

File: civo/datasource_kubernetes_cluster.go [hunk 1]
```diff
@@ -64,8 +64,9 @@ func dataSourceKubernetesCluster() *schema.Resource {
 				Description: "The cni for the k3s to install (the default is `flannel`) valid options are `cilium` or `flannel`",
 			},
 			"tags": {
-				Type:        schema.TypeString,
+				Type:        schema.TypeSet,
```
Comment (RealHarshThakur) [inline civo/datasource_kubernetes_cluster.go:67]: Does Set mean a slice of string?

**Comment by RealHarshThakur** [comment]
Can we please have some docs around acceptance tests? What purpose they'll serve, when they'll be run, etc

**Comment by alejandrojnm** [comment]
In the README.md you can see the propose of the acceptance tests

File: civo/datasource_kubernetes_cluster.go (hunk 1 above)
Comment (alejandrojnm) [reply civo/datasource_kubernetes_cluster.go:67]: I use the typeSet and them I say the Elem is `&schema.Schema{Type: schema.TypeString},`

**Review by RealHarshThakur — APPROVED** [review]

**Pull request merged**